internal/gatewayapi/status/gatewayclass.go

```diff
@@ -17,7 +17,11 @@ import (
 	"time"
 
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/sets"
 	gwapiv1 "sigs.k8s.io/gateway-api/apis/v1"
+	"sigs.k8s.io/gateway-api/conformance/utils/suite"
+
+	"github.com/envoyproxy/gateway/test/conformance"
```

Comment:
it's not a good idea to import a test package here.

Reply:
fixed